src/runtime/dom-particle.ts

```diff
@@ -36,13 +43,13 @@ export class DomParticle extends XenStateMixin(DomParticleBase) {
   /**
    * Override if necessary, to do things when props change.
    */
-  willReceiveProps(...args) {
+  willReceiveProps(...args): void {
```

Comment:
I've been suggesting that we leave return types to type inference, as this is something the typescript compiler can do for us. dom-particle could perhaps be an exception here, though, as it is programmer-exposed.